test/specs/config/node-configuration.js

```diff
@@ -127,6 +127,26 @@ describe('modules/config/node-configuration', function() {
             assert.equal(configuration.getPresetName(), 'jquery');
         });
 
+        it('should load external preset with .jscsrc extension', function() {
+            configuration.registerDefaultRules();
+
+            configuration.load({
+                preset: path.resolve(__dirname + '/../../data/configs/jscsrc/external.jscsrc')
+            });
+
+            var exist = false;
+            configuration.getRegisteredRules().forEach(function(rule) {
```

Comment:
Sure, although it seems logical to do in the different commit, since we use `forEach` in the test above